icgcget/cli.py

```diff
@@ -55,6 +55,38 @@ def logger_setup(logfile):
     return logger
 
 
+def filter_manifest_ids(manifest_json):
+    fi_ids = []
+    for repo_info in manifest_json["entries"]:
+        if repo_info["repo"] in REPOS:
+            for file_info in repo_info["files"]:
+                if file_info["id"] in fi_ids:
+                    logger.error("Supplied manifest has repeated file identifiers.  Please specify a " +
+                                 "list of repositories to prioritize")
+                    raise click.Abort
+                else:
+                    fi_ids.append(file_info["id"])
+    else:
```

Comment:
I need to return the deduplicated Fi ids when a manifest ID is passed into dry run.  It's a bit of a hack, but I don't know any way to get the metadata back without making a second api call.